src/Components/WebAssembly/Server/src/AuthenticationStateSerializer.cs

```diff
@@ -17,6 +17,14 @@ internal sealed class AuthenticationStateSerializer : IHostEnvironmentAuthentica
     private readonly PersistingComponentStateSubscription _subscription;
 
     private Task<AuthenticationState>? _authenticationStateTask;
+    private AuthenticationStateData? _authenticationStateData;
+
+    [PersistentComponentStateStore]
```

Comment:
This attribute does not exist. You wanted to use `[SupplyParameterFromPersistentComponentState]`.